### PR TITLE
Include Movement ecosystem

### PR DESCRIPTION
**Pull request by 0xPrimata**
Include Movement ecosystem
Include Movement as a sub ecosystem of Avalanche, Move, Aptos and Sui

### CODE REVIEW

File: data/ecosystems/a/avalanche.toml [hunk 1]
```diff
@@ -597,8 +598,8 @@ url = "https://github.com/ava-labs/glitch-hackathon-test"
 url = "https://github.com/ava-labs/gnosis-subnet"
 
 [[repo]]
-url = "https://github.com/ava-labs/go-ethereum"
 tags = ["Protocol"]
+url = "https://github.com/ava-labs/go-ethereum"
```
Comment (eherrerosj) [inline data/ecosystems/a/avalanche.toml:602]: can you keep the tags parameter below the URL like it was originally? not sure why it was changed

File: data/ecosystems/a/avalanche.toml [hunk 2]
```diff
@@ -554,8 +555,8 @@ url = "https://github.com/ava-labs/chains"
 url = "https://github.com/ava-labs/chainsafe-scripts"
 
 [[repo]]
-url = "https://github.com/ava-labs/coreth"
 tags = ["Protocol"]
+url = "https://github.com/ava-labs/coreth"
```
Comment (eherrerosj) [inline data/ecosystems/a/avalanche.toml:559]: can you keep the tags parameter below the URL like it was originally? not sure why it was changed

File: data/ecosystems/a/avalanche.toml [hunk 3]
```diff
@@ -487,8 +488,8 @@ url = "https://github.com/ava-labs/avalanche-testing"
 url = "https://github.com/ava-labs/avalanche-types-rs"
 
 [[repo]]
-url = "https://github.com/ava-labs/avalanche-wallet"
 tags = ["Protocol"]
+url = "https://github.com/ava-labs/avalanche-wallet"
```
Comment (eherrerosj) [inline data/ecosystems/a/avalanche.toml:492]: can you keep the tags parameter below the URL like it was originally? not sure why it was changed

File: data/ecosystems/a/aptos.toml [hunk 4]
```diff
@@ -121,8 +122,8 @@ url = "https://github.com/aptos-labs/aptos-ansible"
 url = "https://github.com/aptos-labs/aptos-canvas"
 
 [[repo]]
-url = "https://github.com/aptos-labs/aptos-core"
 tags = ["Protocol"]
+url = "https://github.com/aptos-labs/aptos-core"
```
Comment (eherrerosj) [inline data/ecosystems/a/aptos.toml:126]: can you keep the tags parameter below the URL like it was originally? not sure why it was changed

File: data/ecosystems/s/sui.toml [hunk 5]
```diff
@@ -383,8 +384,8 @@ url = "https://github.com/MystenLabs/snarkjs"
 url = "https://github.com/MystenLabs/solvency-proofs"
 
 [[repo]]
-url = "https://github.com/MystenLabs/sui"
 tags = ["Protocol"]
+url = "https://github.com/MystenLabs/sui"
```
Comment (eherrerosj) [inline data/ecosystems/s/sui.toml:388]: can you keep the tags parameter below the URL like it was originally? not sure why it was changed

File: data/ecosystems/s/sui.toml [hunk 6]
```diff
@@ -604,3 +605,6 @@ url = "https://github.com/xydas97/Move-Workshop-GR"
 
 [[repo]]
 url = "https://github.com/Yureien/Nexus"
+
+[[repo]]
+url = "https://github.com/movemntdev/M1"
```
Comment (eherrerosj) [inline data/ecosystems/s/sui.toml:610]: can remove

**Review by eherrerosj — CHANGES_REQUESTED** [review]
left some comments

**Comment by ashucoder9** [comment]
Hey @eherrerosj, any updates on this PR? We want to make sure Movement Labs is included as a sub-ecosystem of Avalanche.

**Review by eherrerosj — APPROVED** [review]
LGTM, thanks for the contribution @0xPrimata ! Merging

**Pull request merged**